generic3g/specs/WildcardClassAspect.F90

```diff
@@ -45,7 +45,7 @@ module mapl3g_WildcardClassAspect
 
    function new_WildcardClassAspect() result(wildcard_aspect)
       type(WildcardClassAspect) :: wildcard_aspect
-
+      _HERE
```

Comment:
Do we need the debugging print(s)?

Reply:
I think the output from this one may have been buried in the stream of warning diagnostics that I cannot remove yet.
Fixed.